### PR TITLE
Ft import and save many trainees from googlesheet v5

### PR DESCRIPTION
**Pull request by Niyonkuru1**
## What does this PR do?

Import many trainees and their attributes from google sheet and save them into the mongo database

## Description of Task to be completed?

The admin should copy the URL of his / her google sheet and paste them into the input field of the app, then after hitting load, those data should be copied into the database. If the column of the google sheet is not matching the properties in the database collection, he/she should see the error screen with the steps of how to do it correctly. while if it save correctly, he / she will see the screen saying that uploading is successful.

## How should this be manually tested?

- `Clone the repo`

- `checkout` to branch `ft-import-and-save-many-trainees-from-googlesheet-v5`

- Pull the changes by `git pull`

- Install all packages by running `npm install`

- Start app by `running npm run dev`

## Any background context you want to provide?

Note note **you should use only the configured google sheet url, not every google sheet url will work.**

## What are the relevant pivotal tracker stories?

[Trello Board reference](https://trello.com/c/17S8aqBu/58-import-trainees-google-sheet)

### CODE REVIEW

File: src/schema/loadAllTraineesFromGoogleSheet.ts [hunk 1]
```diff
@@ -1,39 +1,41 @@
...
+    gender:String!
+    birth_date:String!
+    Address:String!
+    phone:String!
+    field_of_study:String!
+    education_level:String!
+    province:String!
+    district:String!
+    sector:String!
+    cohort:String!
```
Comment (UmuhireJessie) [inline src/schema/loadAllTraineesFromGoogleSheet.ts]: we no longer use `cohort`. You might consider removing it

File: src/schema/loadAllTraineesFromGoogleSheet.ts [hunk 2]
```diff
@@ -1,39 +1,41 @@
...
+    isEmployed:Boolean!
+    haveLaptop:Boolean!
+    isStudent:Boolean!
+    Hackerrank_score:String!
+    english_score:String!
+    interview:Int!
```
Comment (UmuhireJessie) [inline src/schema/loadAllTraineesFromGoogleSheet.ts:32]: Remove `interview` as well

File: src/schema/resendDataIntoDbTypeDefs.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    education_level: String!
+    province: String!
+    district: String!
+    sector: String!
+    isEmployed: String!
+    haveLaptop: String!
+    isStudent: String!
+    Hackerrank_score: String!
+    english_score: String!
+    interview: String!
```
Comment (UmuhireJessie) [inline src/schema/resendDataIntoDbTypeDefs.ts:25]: `interview` and `interview_decision`? Change `interview` to `interview_score`

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
- Considering the following 

```
{
  "spreadsheetId": "1nltXpm-_7FyIfCwcpd4qYNeXP-tewPRwXaYggRcBDHM"
}
```

- I have this as an output, not sure why `cycle_id` is becoming null, yet that cycle is in my DB

image

**Comment by mugiisha** [comment]
the functionality works well on a successful attempt
**issues**
#### successful message toast on an unsuccessful attempt
pops
#### resend query button not functioning
reseeend
_after a failed attempt one have to refresh the page to try again_

**Review by kareraolivier — CHANGES_REQUESTED** [review]
# squash the commit @Hodal66

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
See comments!

**Pull request merged**